Latex/vorlage/vorlage.tex

```diff
@@ -190,16 +202,16 @@
 
 
 %! PageBreaks nach jeder Section
-\let\oldsection\section
-\renewcommand\section{\clearpage\oldsection}
+\let\oldchapter\chapter
+\renewcommand\chapter{\clearpage\oldchapter}
 
 %! Code-Integration im Dokument
 %? Inklusive Erzeugung eines Custom-Enviroments für Programmcodes
 \usepackage{minted}
 %\SetupFloatingEnvironment{listing}{name=Program code}
 %\SetupFloatingEnvironment{listing}{listname=List of Program Code}
 
-
+\usepackage{scrhack} %necessary to use float with scrreprt
```

Comment:
warum?

Reply:
ich hab doch extra n comment ran gepackt xD
scrrept is an sich nich richtig mit float kompatibel, weil es umgebungen verschieden lustig baut und es sagt einem, wenn man ohne scrhack baut "irgendwas nutzt hier veralteten scheiß, wenn du das brauchst dann binde scrhack ein"

Reply:
ah, ist also ein offizielles ding - wollte nur sicher gehen dass das nicht so ein "fenster nur ankippen sonst bootet meine 2. partition" wird ^^

Reply:
Häää
Verstehe Willys Spruch nicht.